app.drey.PaperPlane.json

```diff
@@ -0,0 +1,232 @@
+            "config-opts": [
+                "-Dbuildtype=release"
+            ],
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler/-/archive/v0.8.1/blueprint-compiler-v0.8.1.tar.gz",
+                    "sha256": "9207697cfac6e87a3c0ccf463be1a95c3bd06aa017c966a7e352ad5bc486cf3c"
+                }
+            ]
+        },
+        {
+            "name": "libsass",
+            "buildsystem": "meson",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/lazka/libsass.git",
+                    "commit": "302397c0c8ae2d7ab02f45ea461c2c3d768f248e"
+                }
+            ]
+        },
+        {
+            "name": "sassc",
+            "buildsystem": "meson",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/lazka/sassc.git",
+                    "commit": "82803377c33247265d779af034eceb5949e78354"
+                }
+            ]
+        },
+        {
+            "name": "gtk",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuildtype=release",
+                "-Dintrospection=disabled",
+                "-Dprint-cups=disabled",
+                "-Ddemos=false",
+                "-Dbuild-testsuite=false",
+                "-Dbuild-examples=false",
+                "-Dbuild-tests=false"
+            ],
+            "cleanup": [
+                "/bin/gtk*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gtk.git",
+                    "commit": "06b3ced8e7fc936caed43379b120d75be09713ca",
+                    "tag": "4.10.3"
+                },
+                {
+                    "type": "patch",
+                    "path": "gtk-reversed-list.patch"
+                }
+            ]
+        },
+        {
+            "name": "appstream",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dbuildtype=release",
+                "-Dstemming=false",
+                "-Dgir=false",
+                "-Dapidocs=false"
+            ],
+            "cleanup": [
+                "/bin/appstreamcli"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/ximion/appstream.git",
+                    "commit": "1ee6af91310f9deb47f3863a7a7ffb22c46bcce6",
+                    "tag": "v0.16.2"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "libyaml",
```

Comment:
if these are only for appstream, then they should both have a "*" cleanup rule.

Reply:
Same here. We'll end with
```
paper-plane: error while loading shared libraries: libyaml-0.so.2: cannot open shared object file: No such file or directory
```